### ISSUE TITLE
the safari extension is not available in my region

### ISSUE THREAD

**Issue by HiKid1098** [pietervanheijningen/clickbait-remover-for-youtube#50]
when I use the link to the safari extension it says it is not available in my region. I am in the mid-west so I have no idea why that keeps happening. It also doesn't show up when I look for it in the App Store.

**Comment by pietervanheijningen** [comment]
Yes unfortunately it has been removed from the App Store, I believe as it did not function correctly.
